hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOmSnapshot.java

```diff
@@ -844,12 +846,12 @@ private String createSnapshot(String volName, String buckName,
     store.createSnapshot(volName, buckName, snapshotName);
     String snapshotKeyPrefix =
         OmSnapshotManager.getSnapshotPrefix(snapshotName);
-    SnapshotInfo snapshotInfo =
-        leaderOzoneManager.getMetadataManager().getSnapshotInfoTable()
-            .get(SnapshotInfo.getTableKey(volName, buckName, snapshotName));
-    String snapshotDirName =
-        OmSnapshotManager.getSnapshotPath(leaderConfig, snapshotInfo) +
-        OM_KEY_PREFIX + "CURRENT";
+    SnapshotInfo snapshotInfo = ozoneManager.getMetadataManager()
+        .getSnapshotInfoTable()
+        .get(SnapshotInfo.getTableKey(volName, buckName, snapshotName));
+    String snapshotDirName = metaDir + OM_KEY_PREFIX +
```

Comment:
Why this particular change?

Reply:
Reverted it to same as it was done previously.